manifests/init.pp

```diff
@@ -87,6 +91,8 @@
   Optional[Integer[1]] $mgmt_pub_interval = $qpid::params::mgmt_pub_interval,
   Optional[Integer[1]] $default_queue_limit = $qpid::params::default_queue_limit,
   Hash[String, Variant[String, Integer]] $custom_settings = $qpid::params::custom_settings,
+  Boolean $service_ensure = true,
+  Boolean $service_enable = true,
```

Comment:
I wonder if it makes sense for us to combine it. For example
```suggestion
  Boolean $service_ensure = true,
  Optional[Boolean] $service_enable = undef,
```

And later use `enable => pick($service_enable, $service_ensure)`

Reply:
For our use case they could be combined, and I'll probably still have our code pass both to be explicit. So unless there is a usecase for supporting one parameter, I'd just leave it at two.

Reply:
In my suggestion you still have them both but you only need to specify one if you want them to match.

Reply:
Interesting idea. I have a few thoughts:

In the proposed usage in puppet-{foreman_proxy_content,katello} we always ensure these parameters have the same value anyway, so I don't think it makes any difference to a foreman-installer user. It would allow us to remove a line of code over there, but might also make things less readable / make the behavior a little less obvious.

So for the folks who are directly using this module, I think it comes down to: is this pattern ubiquitous enough for services that we'd be violating user expectations by not adhering to it? If not, then perhaps we could leave it for now?

Reply:
I should refresh the page before posting comments :)

Reply:
I think it's hard to misuse. In most cases you want them to match anyway. If you don't, you can set them both. If both are explicit parameters, you must remember to set them both. I'd guess that in 99% cases they match and this optimizes for it.